docker-compose.yml

```diff
@@ -1,21 +1,34 @@
 version: "3.4"
 services:
-  lodestar:
-    build: .
+  beacon_node:
+    build: ../lodestar
+    image: lodestar/lodestar
     restart: always
     volumes:
-      - lodestar:/root/.local/share/lodestar
-    command: beacon
+      - beacon_node:/root/.local/share/lodestar
+      - /root/.local/share/lodestar/${NETWORK:-pyrmont}:/root/.local/share/lodestar/${NETWORK:-pyrmont}
```

Comment:
Why is it necessary to map the `beacon_node` lodestar dir to the host's?

Reply:
i'm not sure, this was already there before, i just changed the name from `lodestar` to `beacon_node`

Reply:
but yeah, didn't seem necessary, so i removed it